### PR TITLE
release 1.8.1

### PR DESCRIPTION
**Pull request by stklcode**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -116,6 +117,13 @@ has to be added to the theme's `functions.php`. The condition has modified such
 ## Changelog ##
 You can find the full changelog in [our GitHub repository](https://github.com/pluginkollektiv/statify/blob/master/CHANGELOG.md).
 
+### 1.8.1
+* Fix AMP compatibility for Standard and Transitional mode (#181) (#182)
+* JavaScript is no longer embedded if request is served by AMP (#181) (#182)
+* Always register the action for the cleanup (#184)
+* Exclude sitemap calls (WP 5.5) from tracking (#185) (#186)
```
Comment (florianbrinkmann) [inline readme.txt]: The »(WP 5.5)« might sound like that is something only for WP 5.5, maybe change it to something like »Exclude calls to the sitemap added in WP 5.5 from tracking (#185) (#186)« or so?

**Review by florianbrinkmann — APPROVED** [review]
Looks good to me.
Made a suggestion for a small change in the changelog, but maybe it is just me and it is clear as it is :)

File: readme.txt (hunk 1 above)
Comment (stklcode) [reply readme.txt]: We used `WP 5.3+` in the 1.8.0 changelog. I'd be fine with either your suggestion or the shorthand "(... WP5.5+)".

**Comment by florianbrinkmann** [comment]
»WP5.5+« is good!

**Review by patrickrobrecht — APPROVED** [review]

**Pull request merged**